examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,74 @@
+import { Tree } from "./tree"
+import { IMerkleTree} from "./merkleTree.interface";
+import { PreciseProofs } from "ew-precise-proofs-js";
+
+export class MerkleTree implements IMerkleTree {
+  tree: Tree;
+
+  constructor(document: any){
+    const merkleTree: Tree = this.createTree(document);
```

Comment:
since createTree is catching error and not returning anything this method will return undefined if error occurs during tree creation, which will throw error on lines 11 and 12, maybe modify createTree method to return empty tree in case of an error, or add some if (merkleTree) guard here

Reply:
@biscuitdey i agree with @skosito ... please add. thank u